CHANGELOG.md

```diff
@@ -153,7 +153,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix block explorer link on custom networks for the cases when link is invalid or left empty ([#13870](https://github.com/MetaMask/metamask-extension/pull/13870))
 - Fix signature parsing errors re-surfaced due to 4byte function signature directory being down, by removing the directory([#15300](https://github.com/MetaMask/metamask-extension/pull/15300))
 - Fix intermitent failure when performing a Send tx in non-EIP-1559 networks (like Optimism) by setting the `gasPrice` ([#15628](https://github.com/MetaMask/metamask-extension/pull/15628))
-
+- Fix disabled label colors for Token Symbol and Token Decimal while in dark mode in 'Import Token Page'. ([#15590](https://github.com/MetaMask/metamask-extension/pull/15743))
```

Comment:
This isn't necessary as we will generate the changelog for the release this gets landed in.  We use a tool that sources details of commits in a release to generate the changelog :)

Reply:
Ahh okay! Thanks for letting me know.